server/test/modules/MainModuleTest.java

```diff
@@ -0,0 +1,68 @@
+package modules;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import annotations.BindingAnnotations.Now;
+import com.google.common.collect.ImmutableMap;
+import com.typesafe.config.ConfigFactory;
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import play.Application;
+import play.inject.BindingKey;
+import play.inject.guice.GuiceApplicationBuilder;
+import play.test.Helpers;
+
+@RunWith(JUnitParamsRunner.class)
+public class MainModuleTest {
+
+  private Application app;
```

Comment:
RE Optional: Converted.

RE @Before: Added a brief comment. We initialize the app in each test since we want a distinct configuration for each. The call to Helpers.stop() is necessary to prevent leaking a DB connection that would cause further tests to fail. Having a @Before isn't necessary since we initialize the app in each test method.